tests/custom-style-only.html

```diff
@@ -0,0 +1,79 @@
+<!doctype html>
+<!--
+@license
+Copyright (c) 2014 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+<script>
+WCT = { waitFor: function (cb) { window.addEventListener('HTMLImportsLoaded', cb) } }
+</script>
+<script src="./test-flags.js"></script>
+<script src="../../webcomponents-platform/webcomponents-platform.js"></script>
+<script src="../../es6-promise/dist/es6-promise.auto.min.js"></script>
+<script src="../../template/template.js"></script>
+<script src="../../html-imports/html-imports.min.js"></script>
+<script>
+  WCT = { waitFor: function (cb) { HTMLImports.whenReady(cb) } }
+</script>
+<script src="../../shadydom/shadydom.min.js"></script>
+<script src="../../custom-elements/custom-elements.min.js"></script>
+<script src="../scoping-shim.min.js"></script>
+<script src="../apply-shim.min.js"></script>
+<script>
+if (window.customElements && customElements.polyfillWrapFlushCallback) {
+  // delay definition of custom-style until after template polyfill loads
+  customElements.polyfillWrapFlushCallback(function(cb) {
+    HTMLImports.whenReady(cb);
+  });
+}
+</script>
+<script src="../custom-style-interface.min.js"></script>
+<script src="module/generated/make-element.js"></script>
+<script src="module/generated/custom-style-element.js"></script>
+<script src="../../web-component-tester/browser.js"></script>
+
+<custom-style id="indoc">
+  <style>
+    x-host {
```

Comment:
change this to `div` instead of `x-host` since this isn't a custom element.

Reply:
Done